stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/StreamAttachmentFactories.kt

```diff
@@ -39,7 +40,7 @@ public object StreamAttachmentFactories {
  * @param content - Composable function that allows users to define the content the [AttachmentFactory] will build using any given
  * [AttachmentState].
  */
-public abstract class AttachmentFactory(
+public open class AttachmentFactory @InternalStreamChatApi constructor(
```

Comment:
Do we need to have this as an `open` class? Since it's not going to be extended. Or do we want to keep it like this, just so that people _could_ extend it if they need to (or once the bug is fixed)

Reply:
Yeah, I was thinking that this way we don't break anyone's code if they've already subclassed it (but we make it scary to subclass with the annotation)

Reply:
Works for me!